source/extensions/filters/http/fault/fault_filter.cc

```diff
@@ -434,6 +434,15 @@ void FaultFilter::onDestroy() {
   }
 }
 
+bool FaultFilter::activeFaults() {
+  // Check if there is a delay timer or response rate limiter still applying
+  if (delay_timer_ != nullptr || fault_settings_->responseRateLimit() != nullptr) {
```

Comment:
You can omit the if/else and just do `return delay_time_ != nullptr ...`

Reply:
Updated, thanks.